src/components-examples/cdk/overlay/cdk-overlay-basic/cdk-overlay-basic-example.html

```diff
@@ -0,0 +1,19 @@
+<!-- This button triggers the overlay and is it's origin -->
+<button (click)="isOpen = !isOpen" type="button" cdkOverlayOrigin #trigger="cdkOverlayOrigin">
+  {{isOpen ? "Close" : "Open"}}
+</button>
+
+<!-- This template displays the overlay content and is connected to the button -->
+<ng-template
+  cdkConnectedOverlay
+  [cdkConnectedOverlayOrigin]="trigger"
+  [cdkConnectedOverlayOpen]="isOpen"
+>
+  <ul class="example-list">
+    <li>Item 1</li>
+    <hr />
```

Comment:
The `<hr>` between each item shouldn't be necessary; you could style the items with CSS if you want to show some divider between them

Reply:
Yes. Done